test/conformance/behaviors/README.md

```diff
@@ -0,0 +1,49 @@
+# Behaviors Defining Conformance
+
+The conformance program is intended to answer the question "What is
+Kubernetes?". That is, what features, functions, and APIs are needed in order to
+call something "Kubernetes". Since v1.9, this has been defined as passing a
+specific set of e2e tests.  As part of the [conformance behavior KEP](https://git.k8s.io/enhancements/keps/sig-architecture/20190412-conformance-behaviors.md),
+this instead is moved to an explicit list of "behaviors", which are captured in
+this directory tree. The e2e tests are used to validate whether specific
+behaviors are met, but the definition of conformance is based upon that list of
+approved behaviors. That begs the question, "what is a behavior?".
```

Comment:
I think a section about why we're using behaviors instead of e2e test descriptions would be helpful. Maybe something outlining the differences and benefits of having a behavior separated from an e2e test.

Reply:
Done